cg/store/api/find_basic_data.py

```diff
@@ -90,6 +98,14 @@ def get_bed_version_by_short_name(self, bed_version_short_name: str) -> BedVersi
             filter_functions=[BedVersionFilter.FILTER_BY_SHORT_NAME],
         ).first()
 
+    def get_bed_by_id(self, bed_id: int) -> Bed:
```

Comment:
If we decide to change it to `get_panel_bed`, it would be appropriate to rename the `Bed` class as `PanelBed` as well in my opinion. For the time being, I'll leave it as it is